src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -350,31 +348,31 @@ public enum EvalMetricType
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "Enable categorical split or not.", ShortName = "cat")]
         [TlcModule.SweepableDiscreteParam("UseCat", new object[] { true, false })]
-        public bool? UseCat;
+        public bool? UseCategoricalSplit;
 
-        [Argument(ArgumentType.AtMostOnce, HelpText = "Enable missing value auto infer or not.")]
+        [Argument(ArgumentType.AtMostOnce, HelpText = "Enable special handling of missing value or not.")]
         [TlcModule.SweepableDiscreteParam("UseMissing", new object[] { true, false })]
         public bool UseMissing = false;
```

Comment:
>UseMissing [](start = 20, length = 10)

HandleMissingValue maybe?

Reply:
I will update to HandleMissingValue.

---
In reply to: [261476158](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261476158) [](ancestors = 261476158)